enhancements/node-tuning/containerize-tuned.md

```diff
@@ -0,0 +1,255 @@
+---
+title: containerize-tuned
+authors:
+  - "@yanirq"
+  - "@jmencak"  
+reviewers:
+  - "@MarSik"
+  - "@ffromani"
+  - "@Tal-or"
+  - "@browsell"
+  - "@bwensley"
+  - "@swatisehgal"
+  - "@mrunalp"  
+approvers:
+  - "@MarSik"
+  - "@browsell"
+  - "@mrunalp"
+api-approvers:
+  - "None"
+creation-date: 2024-03-11
+last-updated: 2024-03-14
+tracking-link:
+  - https://issues.redhat.com/browse/PSAP-1236
+---
+
+# Containerize Tuned
+
+## Summary
+
+The proposed enhancement is to restructure cluster-node-tuning-operator (NTO) to run TuneD from a container (e.g. using podman from a systemd service) instead of running in daemon mode so TuneD will initially set defaults and hand off ownership to other services that apply tunings.
+
+## Motivation
+
+With the current implementation of [cluster-node-tuning-operator](https://github.com/openshift/cluster-node-tuning-operator) (NTO) having TuneD running in a daemonset there is an overarching issue where we have multiple services manipulating tuning data - tuned, crio and some system services for irqs where there should only be one.
```

Comment:
This is not the main issue. Multiple ownership is not fixed by moving tuned into podman. What this EP fixes is the late tuning as described below.

Reply:
Its indeed missing this bottom line.  I'll add it .